packages/apidom-ls/src/config/openapi/parameter/lint/schema--mutually-exclusive-openapi-3-1-asyncapi-2.ts

```diff
@@ -0,0 +1,26 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
```

Comment:
Again this linting rule and changes in https://github.com/swagger-api/apidom/pull/3640/files#diff-68bfcc373a81a4cbf60d831b58dd1601e18e8328f93f31f32bd54d683480a438 looks fishy.